samples/drivers/rtc/sample.yaml

```diff
@@ -4,9 +4,16 @@ tests:
   sample.drivers.rtc:
     platform_allow:
       - stm32f3_disco
+    integration_platforms:
+      - stm32f3_disco
```

Comment:
this is only needed if we have a huge number of supported boards, but only want to test some of them regularly, since there is only one allowed, it is redundant :)